### ISSUE TITLE
connect "ECONNREFFUSED" Error.

### ISSUE THREAD

**Issue by prashantgbhangre** [istrategylabs/node-twitter#35]
Hello Guys ,

i am getting error on my remote linux machine as a connect Econnrefused.

Thanks in advance.

**Comment by joesolly** [comment]
Does your remote machine have internet access to twitter.com? Can you ping it from there?

**Comment by prashantgbhangre** [comment]
yes my vm is connected to internet. and ping is also working fine.
